### ISSUE TITLE
Request for new features: Dropout layers and Rename Layer Labels

### ISSUE THREAD

**Issue by numoonchld** [alexlenail/NN-SVG#26]
- It would be great to be able to add dropout layers graphics between hidden, input and output layers in FCNN style drawings

- It would also be nice to be able to rename layers, and add activation labels in FCNN style drawings

I am using Keras to build a neural net, and while the internal graphviz based model visualizer does the job, it does not display an actual neural net schematic. 

Being able to add layers similar to those found [here](https://keras.io/layers/core/) would be helpful
